streams/src/test/java/org/apache/kafka/streams/processor/internals/HighAvailabilityStreamsPartitionAssignorTest.java

```diff
@@ -149,35 +145,12 @@ private void configurePartitionAssignorWith(final Map<String, Object> props) {
         overwriteInternalTopicManagerWithMock();
     }
 
-    // Useful for tests that don't care about the task offset sums
```

Comment:
This is found that after we refactored with mockito, the passed in task set is not needed any more, so I removed them to eliminate the warnings.